pkg/multiplexer/multiplexer.go

```diff
@@ -25,7 +24,7 @@ type Request struct {
 	ctx          context.Context
 	Header       http.Header
 	layoutURL    string
-	fragments    []Fragment
+	fragments    []Requestable
```

Comment:
```suggestion
	requestables    []Requestable
```

maybe multiplexer doesn't need to have a concept of "fragments"?